cache/cache.go

```diff
@@ -151,7 +179,7 @@ func (rl *RedisCache) Throttle(ctx context.Context, rawToken string, l ratelimit
 	}
 
 	// collect quota/burst control and token fetch results
-	tok, err := rl.checkToken(tokenGet, rawToken)
+	tok, _ := rl.checkToken(tokenGet, rawToken)
```

Comment:
We should return the error not swallowing it:
```suggestion
	tok, err := rl.checkToken(tokenGet, rawToken)
	if err != nil {
		return "", err
	}
```